modeshape-parent/pom.xml

```diff
@@ -1850,6 +1851,17 @@
                 <artifactId>ecore-xmi</artifactId>
                 <version>${eclipse.emf.ecorexmi.version}</version>
             </dependency>
+            <!--Video sequencer-->
+            <dependency>
+                <groupId>io.humble</groupId>
+                <artifactId>humble-video-all</artifactId>
+                <version>${humblevideo.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>io.humble</groupId>
+                <artifactId>humble-video-noarch</artifactId>
+                <version>${humblevideo.version}</version>
+            </dependency>
```

Comment:
what's the difference between the 2 artifacts ? Normally an artifact that ends with `-all` should include all existing components.